core/templates/pages/learner-dashboard-page/card-display/card-display.component.html

```diff
@@ -11,15 +11,11 @@
     </div>
   </div>
   <hr color="#00645c" class="mt-2 mb-4" size="2" width="100%">
-  <div class="d-flex card-display-content-container pb-1 mb-n1" #cards>
+  <div class="d-flex card-display-content-container pb-1 mb-n1" [ngClass]="getVisibility()" #cards>
     <ng-content></ng-content>
   </div>
-  <!--TODO(#18384): Implement view more for progress tab (styling)-->
-  <!--TODO(#18384): Hide when there's enough space for all cards etc-->
-  <button *ngIf="!tabType.includes('home')" class="align-self-end card-display-more-button mt-4">
-    {{ 'I18N_LEARNER_DASHBOARD_CARD_DISPLAY_BUTTON' | translate }}
-    <span class="fas fa-chevron-down ml-2"></span>
-  </button>
+  <!--TODO(#18384): Delete I18N_LEARNER_DASHBOARD_CARD_DISPLAY_BUTTON key-->
+  <oppia-content-toggle-button *ngIf="!tabType.includes('home') && ((numCards * cardWidth) - 16 > cards.offsetWidth)" (toggleUpdated)="handleToggleState($event)"> </oppia-content-toggle-button>
```

Comment:
I'm having trouble understanding this -- what does 16 represent, and why 16? Also, do you think it would be better to move this computation to a function?

Reply:
The cardWidth accounts for the 16px gap between each card (width: 216px). This means we consider an extra gap at the end, which causes the display more button to appear even when we have enough space for the cards. 

Moved computation to a function. Done.